screens/metricTank

```diff
@@ -20,9 +20,7 @@ cd /go/src/github.com/raintank/raintank-metric/metric_tank
 	--listen :6063 \
 	--primary-node \
 	--elastic-addr elasticsearch:9200 \
-	--redis-addr redis:6379 \
 	--elastic-warmup-pct 100 \
-	--cassandra-addrs cassandra &>> /var/log/raintank/nsq_metrics_tank.log &
-tail -f /var/log/raintank/nsq_metrics_tank.log 
-
+	--cassandra-addrs cassandra \
+|& tee /var/log/raintank/nsq_metrics_tank.log 
```

Comment:
it's best to always redirect to a file and then tail that file. this means that when the stack is open, you can always kill the tail, do whatever you want to do, and go back in history ( up arrow) and tail again.
it's a useful workflow pattern because often i want to inspect certain things within a specific container.